src/base/perf_monitor.c

```diff
@@ -1071,7 +1071,7 @@ perfmon_get_from_statistic (THREAD_ENTRY * thread_p, const int statistic_id)
       return 0;
     }
 
-  if (thread_p->m_px_orig_thread_entry != NULL && thread_p->m_px_orig_thread_entry != thread_p)
+  if (thread_p->m_px_orig_thread_entry && thread_p->m_uses_px_stats)
     {
```

Comment:
```suggestion
  if (thread_p != NULL && thread_p->m_uses_px_stats)
    {
      assert (thread_p->m_px_orig_thread_entry != NULL);

```
`src/base/perf_monitor.h`와 같은 변경사항입니다.

Reply:
https://github.com/CUBRID/cubrid/pull/6442/commits/45a768b7ecc716fcd40549a247c4a9dbbf6438cf 에서 적용했습니다.

Reply:
실수로 `m_px_orig_thread_entry `를 설정하지 않는 경우에 대해서 검증할 수 있는 방법이 있는게 좋지 않을까요?

Reply:
https://github.com/CUBRID/cubrid/pull/6442/commits/7b58c96528bc4b6a7071d3db321b11b092d5731b 에서 assert를 추가하였습니다.